tests/package.json

```diff
@@ -5,24 +5,23 @@
   "main": "index.html",
   "scripts": {
     "generate": "run-p generate:*",
-    "orval": "node ../packages/orval/dist/bin/orval.js",
-    "generate:cli": "yarn orval --input ./specifications/petstore.yaml --output ./generated/cli/petstore/endpoints.ts",
-    "generate:default": "yarn orval --config ./configs/default.config.ts",
-    "generate:axios": "yarn orval --config ./configs/axios.config.ts",
-    "generate:angular": "yarn orval --config ./configs/angular.config.ts",
-    "generate:react-query": "yarn orval --config ./configs/react-query.config.ts",
-    "generate:svelte-query": "yarn orval --config ./configs/svelte-query.config.ts",
-    "generate:vue-query": "yarn orval --config ./configs/vue-query.config.ts",
-    "generate:swr": "yarn orval --config ./configs/swr.config.ts",
-    "generate:multi-file": "yarn orval --config ./configs/multi-file.config.ts",
-    "generate:zod": "yarn orval --config ./configs/zod.config.ts",
+    "@vetster/orval": "node ../packages/orval/dist/bin/orval.js",
```

Comment:
Do we want to rename the scripts? Seems unnecessary to me

Reply:
I replaced every `orval` with `@vetster/orval.`. I can revert the script name if that works.